src/main/java/frc/robot/RobotContainer.java

```diff
@@ -45,10 +45,10 @@ public RobotContainer() {
   }
  
   private void configureBindings() {
-    m_driverController.btn_B.whileTrue(com_IntakeGround);
     m_driverController.btn_X.whileTrue(com_PrepShooter);
     m_driverController.btn_A.whileTrue(com_StageGP);
     m_driverController.btn_LeftBumper.whileTrue(com_IntakeHopper);
+    m_driverController.btn_LeftBumper.whileTrue(com_IntakeGround);
```

Comment:
That works but you can also do this :)
```suggestion
    m_driverController.btn_LeftBumper.whileTrue(com_IntakeHopper).whileTrue(com_IntakeGround);
```